sql/7actualActivityForm.sql

```diff
@@ -0,0 +1,31 @@
+--
+-- Table structure for table `dbActualActivityForm`
+--
+
+CREATE TABLE `dbActualActivityForm` (
+  `id` INT AUTO_INCREMENT PRIMARY KEY,
+  `activity` INT NOT NULL,
+  `date` VARCHAR(256) NOT NULL,
+  `program` VARCHAR(256) NOT NULL,
+  `start_time` VARCHAR(15) NOT NULL,
+  `end_time` VARCHAR(15) NOT NULL,
+  `start_mile` VARCHAR(15) NOT NULL,
+  `end_mile` VARCHAR(15) NOT NULL,
+  `address` VARCHAR(256) NOT NULL,
+  `attend_num` INT NOT NULL,
+  `volstaff_num` INT NOT NULL,
+  `materials_used` TEXT NOT NULL,
+  `mealinfo` BOOLEAN NOT NULL,
+  `act_costs` TEXT NOT NULL,
+  `act_benefits` TEXT NOT NULL,
```

Comment:
You've got an extra comma here thats throwing an error, just remove the last comma on the act_benefits line
![image](https://github.com/user-attachments/assets/099f6301-0e16-41b6-ae16-4d64c77d674b)

Reply:
fixed!